src/js/utils/ServiceUtil.js

```diff
@@ -28,6 +28,27 @@ const ServiceUtil = {
         definition.instances = formModel.General.instances;
       }
 
+      if (formModel.Optional != null) {
+        definition.executor = formModel.Optional.executor;
+        definition.fetch = formModel.Optional.uris &&
+          formModel.Optional.uris.split(',')
```

Comment:
Accepted will change, and also do that for the General part above. Should I do that in this PR or do you prefer if we do that in another PR?